src/containers/Appointment.js

```diff
@@ -0,0 +1,91 @@
+import { Link, useLocation } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import userAuth from '../helpers/userAuth';
+import { useHttp } from '../hooks/http';
+import { loadAppointment } from '../actions';
+import { formatDate, formatTime } from '../helpers/time';
+import MedicalImg1 from '../images/medical_care.svg';
+import MedicalImg2 from '../images/medicine.svg';
+
+const Appointment = () => {
+  const location = useLocation();
+
+  const loggedIn = useSelector((state) => state.user);
+  const appointment = useSelector((state) => state.appointment);
+  const userInfo = JSON.parse(localStorage.getItem('user'));
+
+  if (loggedIn || userAuth()) {
+    const { id } = location.state;
+
+    useHttp(
+      `https://digidocs-api.herokuapp.com/api/v1/${userInfo.role}/${userInfo.data.id}/appointments/${id}`,
```

Comment:
Do you think it's better to hard code URL's? What if in the future we purchase a domain name and we change this URL?